openshift/template.yaml

```diff
@@ -65,7 +65,7 @@ objects:
     jobTemplate:
       spec:
         ttlSecondsAfterFinished: 10800 # keep last 3 runs (3 * 1h)
-        backoffLimit: 3
+        backoffLimit: 4
```

Comment:
If we parametrize this value we can change it directly from saas file, thus only needing one change and no code change instead of potentially two and potentially forcing the deploy of the latest software

Reply:
This is actually the purpose, I need to get the pod to restart on stage

Reply:
If I understand you correctly

Reply:
What I've suggested is instead of hardcoding the value here, we could set something like `backoffLimit: ${{BACKOFF_LIMIT}}`.

The advantage of this is not immediate, as in this instance the process is the same, but for all future changes of the same value, especially for saas file targets with a reference hash. The reason is that this way we 1) are not forced to deploy new software when we really just want to change the configuration 2) we will perform only one change even when the target has a reference hash (as opposed as one change here and one to update the ref)